data-serving/scripts/aggregate/aggregate/app.py

```diff
@@ -210,10 +217,11 @@ def generate_country_json(cases, s3_endpoint, bucket, date, line_list_url, map_u
             record["lat"] = country_codes[country]['latitude']
             record["long"] = country_codes[country]['longitude']
         except KeyError:
-            raise ValueError(f"Could not find country in list of country codes: {country}")
+            logging.critical(f"Could not find country in list of country codes: {country}")
```

Comment:
Is this critical? The script ignores the error, maybe it's just a warning (or the script should do more)?

Reply:
Before it was failing on countries with relatively low numbers of reported cases, so it seemed like the right thing to do was to let the script continue.